src/main/java/emu/grasscutter/server/game/GameServer.java

```diff
@@ -269,6 +269,8 @@ public void run() {
 	@Override
 	public void onStartFinish() {
 		Grasscutter.getLogger().info(translate("messages.status.free_software"));
+		Grasscutter.getLogger().info(translate("messages.status.game_version"));
+		Grasscutter.getLogger().info(translate("messages.status.version"));
```

Comment:
Do something like this but edit it to use the new translation system. 
https://github.com/mingjun97/Grasscutter/blob/5aefad61eacfad17c46a4f139b5d5c8f56c0aff7/src/main/java/emu/grasscutter/Grasscutter.java#L84
(Originally from an old PR which was merged but was accidently removed)

Reply:
Thanks for the help, I've fiddled around with the code a little more and everything works perfectly on my side!